apps/arena/lib/arena/game_updater.ex

```diff
@@ -79,7 +84,11 @@ defmodule Arena.GameUpdater do
 
           player ->
             health = max(player.aditional_info.health - projectile.aditional_info.damage, 0)
-            player = put_in(player, [:aditional_info, :health], health)
+
+            player =
+              put_in(player, [:aditional_info, :health], health)
+              |> put_in([:aditional_info, :last_natural_healing_update], now)
```

Comment:
Don't follow this, why do we need it?

Reply:
Because we will heal players only after some interval, so we need to know when was the last time we healed it.
Do you have another way to do it?

Reply:
Didn't understand why we needed because of the naming jajaja got it now, all good